frontends/web/src/utils/markup.ts

```diff
@@ -17,16 +17,25 @@
 
 import { h, JSX } from 'preact';
 
-type SimpleMarkupProps = {
+type MarkupProps = {
     tagName: keyof JSX.IntrinsicElements;
     markup: string;
 } & JSX.HTMLAttributes;
 
 const captureStrongElement = /^(.*)<strong>(.*)<\/strong>(.*)$/;
 
-// SimpleMarkup renders `foo <strong>bar</strong> baz` safely as `foo <strong>bar</strong> baz`. Anything else is rendered as sanitized text.
-// Only <strong> is supported to keep it simple.
-export function SimpleMarkup({ tagName, markup, ...props }: SimpleMarkupProps) {
+/**
+ * **SimpleMarkup** renders `foo <strong>bar</strong> baz` safely as
+ * `foo <strong>bar</strong> baz`. Anything else is rendered as
+ * sanitized text.
+ * Only one occurence of <strong> is supported to keep it simple.
+ * ### Example:
+ * ```jsx
+    <SimpleMarkup tagName="p" markup="foo <strong>bar</strong> baz" />
+ * ```
+ */
+
```

Comment:
remove this blankline since the comment is supposed to be attached to the function? and the same for `multilineMarkup` below.

Reply:
done